merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/models/MissionModelTest.java

```diff
@@ -47,18 +46,20 @@ public void shouldGetActivityTypeList() {
                     new Parameter("y", ValueSchema.STRING),
                     new Parameter("vecs", ValueSchema.ofSeries(ValueSchema.ofSeries(ValueSchema.REAL)))),
                 List.of(),
-                new EnumValueMapper<>(Unit.class).getValueSchema()
+                new UnitValueMapper().getValueSchema()
             ));
 
         // WHEN
         final var activityTypes = new HashMap<String, ActivityType>();
-        registry.taskSpecTypes().forEach((name, specType) -> {
-            activityTypes.put(name, new ActivityType(name, specType.getParameters(), specType.getRequiredParameters(), specType.getReturnValueSchema()));
-        });
-        final Map<String, ActivityType> typeList = activityTypes;
-
-        // THEN
-        assertThat(typeList).containsAllEntriesOf(expectedTypes);
+        registry.taskSpecTypes().forEach((name, specType) -> activityTypes.put(
+            name,
+            new ActivityType(name,
+                             specType.getParameters(),
+                             specType.getRequiredParameters(),
+                             specType.getReturnValueSchema())));
+
+      // THEN
+        assertThat(activityTypes).containsAllEntriesOf(expectedTypes);
```

Comment:
Slightly wonky indentation here. (Also, this block is a pure-whitespace change, so I had to squint to make sure it wasn't actually changing anything amidst the other changes.)

Reply:
I also changed `-> { activityTypes` to `-> activityTyes` but very tiny refactor let me change that weird whitespace issue.

Reply:
I also removed some unused imports :P

Reply:
Ah, true. I guess I'd consider that a formatting change (so, still semantically inert), but you're right that it's not whitespace-only. Thanks for highlighting that extra change for me.

Reply:
> I also removed some unused imports :P

Yeah, that's fine, I was commenting on the small block of changes around the `// THEN` in this instance :)